Cargo.toml

```diff
@@ -10,6 +10,7 @@ keywords    = ["tags", "audio", "metadata", "id3", "vorbis"]
 categories  = ["multimedia", "multimedia::audio", "parser-implementations"]
 readme      = "README.md"
 include     = ["src", "Cargo.toml", "LICENSE-APACHE", "LICENSE-MIT", "SUPPORTED_FORMATS.md"]
+rust        = "1.70.0"
```

Comment:
```suggestion
```
I'd rather not make any guarantees about the MSRV yet. I always just use the latest stable, so this is something extra I'd have to remember to bump on each release.